packages/fluent/scss/listview/_layout.scss

```diff
@@ -18,6 +18,14 @@
         position: relative;
         -webkit-touch-callout: none;
         -webkit-tap-highlight-color: transparent;
+
+        &.k-disabled {
+            & > .k-listview-header,
+            & > .k-listview-footer,
+            & > .k-listview-pager {
+                color: inherit;
+            }
+        }
```

Comment:
These styles are not needed. The ListView component API does not provide a way to disable the component.